config/eloquent-driver.php

```diff
@@ -18,10 +18,12 @@
     ],
 
     'collections' => [
-        'driver'     => 'eloquent',
-        'model'      => \Statamic\Eloquent\Collections\CollectionModel::class,
-        'tree'       => \Statamic\Eloquent\Structures\CollectionTree::class,
-        'tree_model' => \Statamic\Eloquent\Structures\TreeModel::class,
+        'driver'       => 'eloquent',
+        'model'        => \Statamic\Eloquent\Collections\CollectionModel::class,
+        // Uncomment the setting below if you need to specify a different driver for your collections tree.
+        // 'tree_driver' => 'eloquent',
```

Comment:
This should be the default to ensure backwards compatibility for existing eloquent driver users. Can you update?

I also wonder if it might be 'cleaner' to have a new config for `collection_trees` rather the bundling this under collections?